doc/conf.py

```diff
@@ -133,6 +133,18 @@
 # Plot directive configuration
 # ----------------------------
 
+plot_pre_code = '''
+# Default imports from plot_directive sphinx extension.
+import numpy as np
+from matplotlib import pyplot as plt
+
+# Initialize random number generator for consistent plots.
+import random
+random.seed(20021210)
+del random
```

Comment:
I don't know for certain whether any of our examples use `random`, but I don't think `np.random` shares seed with it. I delete it because our documentation already mysteriously depends on `np` and `plt` existing and I don't want anyone else to start depending on `random`, too.